examples/config.sample

```diff
@@ -98,9 +98,11 @@
 # Mandatory, no default value
 #Device = 
 
-# Baudrate of the device
+# List of baudrates to try to connect with the device. A default baud rate of
+# 115200 baud will always be added to the end of the list.
```

Comment:
humn... where is this coming from? If user configured the baud as 57600 we should never try to switch to 115200

Reply:
It looks like a regression caused by 46ce50943667 ("Config: Move option tables to endpoint classes (fixes #88)"). We should only append the default baud rate if none is set.

Reply:
It was in the original `parse_baudrates` function, which I only moved to the class and haven't changed (see https://github.com/mavlink-router/mavlink-router/commit/46ce50943667b762b4b09a6acbe5f8cbc7e326bb#diff-34d21af3c614ea3cee120df276c9c4ae95053830d7f1d3deaf009a4625409ad2L535 (in main.cpp)). But you're right, that it doesn't make sense to append it to the list, if it was explicitly configured by the user.

Reply:
yeah... it was some other commit then. I'm pretty sure this worked as intended before as we used that on the Aero drone. Looking at my change in 2f62ec7bfe3d220240b42b2a3d920d3fd684a53b, that one looks correct. So it was probably something in between.

Reply:
I'll change that in this PR shortly. So we'll have default value or list as configured.